nbdime/args.py

```diff
@@ -465,6 +465,12 @@ def add_merge_args(parser):
         default=True,
         help="disallow deletion of transient data such as outputs and "
              "execution counts in order to resolve conflicts.")
+    parser.add_argument(
+        '--no-base',
+        dest='show_base',
+        action="store_false",
+        default=True,
+        help="Don't display the base version.")
```

Comment:
Please add this in `nbdime/webapp/nbmergeweb` instead. It makes no sense for this argument to show up in the CLI command `nbmerge`